### ISSUE TITLE
metadata.yaml really shouldn't define anything

### ISSUE THREAD

**Issue by lazypower** [mbruzek/layer-storage#1]
As this layer is intended to be inherited from, I don't know that its proper to define anything other than the storage pool, and even then I'm hesitant to blindly include a disk option.

This adds a hard requirement of the storage provider to implement what is defined here, what I feel is a better option is to move and document the required additions to the inheriting `metadata.yaml`

**Comment by marcoceppi** [comment]
I disagree, layer storage should not define any storage, that should be implemented by the upper layer author

**Comment by marcoceppi** [comment]
basically, metadata should be empty

**Comment by lazypower** [comment]
In so many words, thats what I was suggesting ;)

**Comment by marcoceppi** [comment]
duh :+1:
